epochStart/shardchain/trigger.go

```diff
@@ -214,7 +214,7 @@ func (t *trigger) updateTriggerFromMeta(metaHdr *block.MetaBlock, hdrHash []byte
 
 	for hash, meta := range t.mapEpochStartHdrs {
 		canActivateEpochStart := t.checkIfTriggerCanBeActivated(hash, meta)
-		if canActivateEpochStart && t.epoch != meta.Epoch {
+		if canActivateEpochStart && t.epoch+1 == meta.Epoch {
```

Comment:
why was this changed to t.epoch+1 == meta.Epoch, was it possible that shard would trigger when it was behind meta with more than one epoch? or ahead? 
otherwise t.epoch != meta.Epoch should have been ok as well.

Reply:
in tests, created scenarios when shard is with 2-3 epoch behind and wants to catch up.

Reply:
add a variable/config param instead of a magic number so we know what it is.

Reply:
it is not in config. as you want to go from epoch to epoch, so the change has to be plus 1.